libclc/ptx-nvidiacl/libspirv/math/cos.cl

```diff
@@ -0,0 +1,26 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "../../../generic/lib/clcmacro.h"
+#include "../../include/libdevice.h"
+#include <spirv/spirv.h>
+
+#define __CLC_FUNCTION __spirv_ocl_cos
+#define __CLC_BUILTIN __nv_cos
+#include "unary_builtin.inc"
+
+#define __CLC_FUNCTION __spirv_ocl_cosh
+#define __CLC_BUILTIN __nv_cosh
+#include "unary_builtin.inc"
+
+/*
+Linking globals named '_Z17__spirv_ocl_cospif': symbol multiply defined!
```

Comment:
can you remove the dead code please ?

Reply:
Done.
For the future, do you think we should keep track of what we have and what's left to do in some file?
like in [cuda-vs-opencl-math-builtin-precisions](sycl/doc/cuda/cuda-vs-opencl-math-builtin-precisions.md)

Reply:
I have them locally, I'm finishing the patch formatting (some are already opened). There is also binding test that were added.